lib/resque_bus/server.rb

```diff
@@ -22,6 +22,123 @@ def self.included(base)
         
       }
     end
+
+    class Helpers
+      class << self
+        def parse_query(query_string)
```

Comment:
Could this move to QueueBus so that it can be shared between sidekiq-bus and resque-bus?

Reply:
sounds like a plan to me.